src/store/account/account.store.impl.ts

```diff
@@ -28,8 +28,9 @@ export class AccountStoreImpl extends StoreImpl implements AccountStore {
       address: observable,
       balance: observable,
       _web3User: observable,
+      userExists: observable,
+      userSignedIn: observable,
```

Comment:
This is not observable. Functions can only be actions if they modify any observables. Please delete both the lines.

Reply:
Got it. Since it is a function whose values need to be observed. Shall we use `computed` with a getter function?